configs/xlml/tensorflow/solutionsteam_tf_nightly_supported_config.py

```diff
@@ -19,6 +19,7 @@
 from configs.xlml.tensorflow import common
 from airflow.models import Variable
 from configs.vm_resource import TpuVersion, Project, RuntimeVersion
+from typing import List
```

Comment:
Nit: we should setup linters/formatters when we have a chance. This type of import should typically be at the top of the module.

Reply:
Good quesiotn, Eric! We do have formatters using [pyink](https://github.com/google/pyink), and check is integrated [here](https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/scripts/format-codes.sh#L24). But I am not sure why it does not report the issue.